internal/utils/templates/init_config.toml

```diff
@@ -60,3 +60,10 @@ enable_confirmations = false
 enabled = false
 client_id = ""
 secret = ""
+
+[value_file]
+path = "./.env.local"
+type = "env"
+anon_key = "NEXT_PUBLIC_SUPABASE_ANON_KEY"
+service_role_key = "SUPABASE_SERVICE_ROLE_KEY"
+api_url = "NEXT_PUBLIC_SUPABASE_URL"
```

Comment:
This stanza actually deals with two separate concerns: setting the export path which applies to all services and configuring individual service keys. I'm thinking the following alternative instead:
- set `anon_key` and `service_role_key` in [auth] stanza
- set `api_url` in [api] stanza
- remove the rest of [value_file] stanza

The last point is to avoid introducing additional CLI flag. Instead of pretty printing the connection details in `utils.ShowStatus`, we can write to stdout in env var format. For example,

```
$ supabase start
Started supabase local development setup.

API_URL=http://localhost:54321
DB_URL=postgresql://postgres:postgres@localhost:54322/postgres
STUDIO_URL=http://localhost:54323
INBUCKET_URL=http://localhost:54324
ANON_KEY=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJzdXBhYmFzZS1kZW1vIiwicm9sZSI6ImFub24ifQ.625_WdcF3KHqz5amU0x2X5WWHP-OEs_4qj0ssLNHzTs
SERVICE_ROLE_KEY=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJzdXBhYmFzZS1kZW1vIiwicm9sZSI6InNlcnZpY2Vfcm9sZSJ9.vI9obAHOGyVVKa3pD--kJlyxp-Z2zV9UUMAhKpNLAcU
```

Users can then redirect stdout to a file for setting up code editors: `supabase start > .env.local`

Other logs will go to stderr so they won't be redirected to env file. @soedirgo

Reply:
Ok, I can make those changes,

Also the names of the env mapping should probably be changed too, like `anon_key_env_var`?

Reply:
Ok, I have updated `utils.ShowStatus()` however the docker pull logs are written to `stdout` instead of `stderr` so when you run `supabase start > .env.local` has all the docker pull logs and stuff.

Reply:
Yup, that seems fine for now. I will take a look at the docker code tmr to pipe those logs to `stderr`. Appreciate your help in driving this change.